config/application.rb

```diff
@@ -227,6 +227,7 @@ class Application < Rails::Application
 
   def self.setup_console
     ENV['IRBRC'] = File.expand_path('../irbrc', __FILE__)
-    puts "For some operations a user must be set, try User.current = User.first"
+    User.current = User.anonymous_console_admin
+    puts "Current user is #{User.current.login}, call User.current= to change it"
```

Comment:
could we also add a warning here? using this might break your warranty kind of warning? :)